src/caughtup/__tests__/caughtUpReducers-test.js

```diff
@@ -113,4 +113,80 @@ describe('caughtUpReducers', () => {
 
     expect(newState).toEqual(expectedState);
   });
+
+  describe('verify that server has send extra message before calculating adjustment', () => {
```

Comment:
When does the server not send an extra message?

Reply:
For the initial fetch of home view it is returning 1 more.
<img width="270" alt="screen shot 2017-11-05 at 12 01 54 pm" src="https://user-images.githubusercontent.com/18511177/32412559-9eb21c0c-c221-11e7-8825-f2e10afbb145.png">

But for the initial fetch of any other narrow it returns same as per requested.
<img width="333" alt="screen shot 2017-11-05 at 12 04 00 pm" src="https://user-images.githubusercontent.com/18511177/32412564-c19fc9d0-c221-11e7-94d3-d9f1ad899778.png">


It better to dynamically calculate rather than hard coding :)